app/controllers/api/redhat_cloud_service_providers_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  class RedhatCloudServiceProvidersController < BaseController
```

Comment:
The .jsx files have red_hat in the name and this is redhat.  I think we should be consistent on red_hat.

Reply:
ok, so updating the API endpoint to /api/red_hat_cloud_service_providers then ?